spec/superbot/capybara/convert_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "spec_helper"
+
+RSpec.describe Superbot::Capybara::Convert do
+  describe "call" do
+    it 'converts json to capybara syntax' do
+      json = [
+        { id: 1, type: 'visit', url: 'http://www.example.com' },
+        { id: 2, type: 'click', selector: 'Link' },
+        { id: 3, type: 'scroll', amountPercent: 10 },
+        { id: 4, type: 'resolution', resolution: [1920, 1080] }
+      ].to_json
+
+      converted_json = described_class.call(json)
+
+      expect(converted_json).to eq(
```

Comment:
it should run these with capybara, not assert strings - it will be slower, but it stays true to the e2e spirit. it is okay if we don't verify the action - but atleast that the converted string does not raise etc.